test/actions/cohorts.spec.js

```diff
@@ -19,3 +24,28 @@ describe('getCohortUsers', () => {
     expect(getCohortUsers({ cohort: 'something' })).toMatchSnapshot();
   });
 });
+
+describe('getCohortProjects', () => {
+  it('should be a function', () => {
+    expect(typeof getCohortProjects).toBe('function');
+  });
+
+  it('should create an appropriate action', () => {
+    expect(getCohortProjects({ cohort: 'someone' })).toMatchSnapshot();
+  });
+});
+
+describe('addCohortProject', () => {
+  it('should be a function', () => {
+    expect(typeof addCohortProject).toBe('function');
+  });
+
+  it('should create an appropriate action', () => {
```

Comment:
lets get simple in our wording 

```suggestion
  it('should create an action', () => {
```